services/core-api/tests/now_application_factories.py

```diff
@@ -363,7 +363,7 @@ class Params:
 
     now_application_id = factory.SelfAttribute('now_application.now_application_id')
     #application_progress_id = factory.Sequence(lambda n: n)
-    application_progress_status_code = factory.LazyFunction(RandomNOWProgressStatusCode)
+    application_progress_status_code = 'REV'
```

Comment:
Is this a limitation of the status code workflow checks that we've put in?

Reply:
there should only be one of each, and they should be ordered. so the first one is REV.. I think we should look at better ways of sequencing this.

Reply:
I agree with setting it to rev. Having it be random caused mismatched state permutations that adding more complex logic to define doesn't seem useful.